app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  def increment_likes
+    post.increment!(:LikesCounter)
+  end
```

Comment:
- [ ] The Like should belong to an `:author`, not `:user`. Kindly fix that.

  ```
  belongs_to :author, class_name: 'User'
  ```

- [ ] Kindly run the necessary migrations as well for the likes table to reference the `author`, not `user`.

- **[OPTIONAL]** I think it would be better to implement a callback for this `increment_likes` method so that it is triggered whenever a post is saved.

  ```
  after _save :increment_likes
  
  private

  def increment_likes
    post.increment!(:LikesCounter)
  end
  ```